plugins/poll/assets/javascripts/controllers/poll-breakdown.js.es6

```diff
@@ -15,6 +16,15 @@ export default Controller.extend(ModalFunctionality, {
   highlightedOption: null,
   displayMode: "percentage",
 
+  @discourseComputed("model.poll.title", "model.post.topic.title")
+  title(pollTitle, topicTitle) {
+    if (pollTitle) {
+      return htmlSafe(pollTitle);
+    }
+
+    return topicTitle;
```

Comment:
That would be a perfect case for using the ternary operator.

```suggestion
    return pollTitle ? htmlSafe(pollTitle) : topicTitle;
```

Also, why does the `pollTitle` require `htmlSafe` and not `topicTitle`?

Reply:
`pollTitle` inherits all properties of markdown heading, like ability to add bold/italic sections, links, inline code blocks, etc, so it emits full html. Topic titles are more restricted as they are not parsed at all.